moduleframework/mtf.yaml

```diff
@@ -0,0 +1,104 @@
+# default module type in case you don't wath everytime use MODULE=docker or define it in config.yaml
+default_module: docker
+
+# openshift specific section
+openshift:
+  init_wait: 50
+  docker_registry: "docker-registry"
+  project: 'project'
+  template: 'template'
+  ip:
+  user:
+  password:
+  local_ip: "127.0.0.1"
+  local_user: "developer"
+  local_password: "developer"
+
+# docker specific section
+docker:
+# default name of Docker file
+  dockerfile: "Dockerfile"
+# default name of HeplMD file
+  helpmdfile: "help.md"
+# default location of these files (based on axiom that you are in tests/ subdir)
+  dockerfiledefaultlocation: "../"
+
+# modularity specific  section
+modularity:
+  tempmodulefile: "tempmodule.yaml"
+  default_module_stream: "master"
+  default_profile: "default"
+
+# systemd-nspawn config
+nspawn:
+# default location where images (directories) lives
+  basedir: "/opt"
+
+# generic section mainly contains timeouts
+generic:
+# default architecture
+  arch: "x86_64"
+# command to find which packager to use (yum, dnf, apt-get)
+  packager_cmd: "test -e /usr/bin/dnf && echo 'dnf -y' ||
+   ( test -e /usr/bin/microdnf && echo 'microdnf'      ||
+   ( test -e /usr/bin/yum && echo 'yum -y'             ||
+   echo 'apt-get -y' ) )"
+# which values are considered as true
+  true_values: ['yes', 'YES', 'yes', 'True', 'true', 'ok', 'OK']
```

Comment:
This can be one general function. I don't see any reason, why to have it here, in config file.

Reply:
make sense